starknet_py/tests/e2e/docs/guide/test_cairo1_contract.py

```diff
@@ -26,7 +26,7 @@ async def test_cairo1_contract(
     from starknet_py.common import create_casm_class
     from starknet_py.hash.casm_class_hash import compute_casm_class_hash
 
-    # contract_compiled_casm is the output of the starknet-sierra-compile (.casm file)
+    # contract_compiled_casm is a string with the output of the starknet-sierra-compile (.casm file)
```

Comment:
```suggestion
    # contract_compiled_casm is the content of the starknet-sierra-compile output (.casm file)
```
Would something like this be clearer maybe?

Reply:
Sounds nicer, but I wanted to put emphasis on the fact that it is a `string` so someone doesn't try to use a path or something else.

Reply:
> > Maybe we should just create some `read_contract` function in public api? Then we could use it in the test.
> 
> Pathlib already provides `.read_text` which is simple enough. Don't see any benefit exposing something like that on our side.

Then I think we should just use the function from pathlib directly in the test (and make it visible in the docs of course).